.azure/pipelines/ci.yml

```diff
@@ -112,6 +112,16 @@ jobs:
             $(_BuildArgs)
       displayName: Build Installers
 
+    - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+      - task: NuGetCommand@2
+        displayName: Push Visual Studio NuPkgs
+        inputs:
+          command: push
+          packagesToPush: '/artifacts/packages/**/NonShipping/VS.Redist.Common.*.nupkg'
+          nuGetFeedType: external
+          publishFeedCredentials: 'DevDiv - VS package feed'
```

Comment:
@johnbeisner what tokens are needed to access this feed? (I can add this task directly to the release pipeline but would rather get it right the first time.)

Reply:
AFAIK, none - the access token is in the communication service.

@jcagme can you confirm?

Reply:
Hmm, do we need a new Service Connection to get this feed (credentials) name working?

Reply:
> @jcagme can you confirm?

Yes, that's stored in the Service Connection.

> Hmm, do we need a new Service Connection to get this feed (credentials) name working?

Can you just reuse the same Service Connection instead of creating a new one? (sorry don't have any context around this change)

Reply:
We need a 'Service Connection' to push to the VS feed. From dnceng-internal to devdiv-internal; 'DevDiv - VS package feed' is the dnceng-internal service connection that bridges that gap. We have this working in the core-sdk and core-setup dnceng-internal daily builds. My understanding is that the access token is in the service connection. @jcagme can confirm the access method and assist you in setting this up in the release pipeline.

Reply:
Our release pipeline is AspNetCore-cd-official (https://dev.azure.com/dnceng/internal/_releaseDefinition?definitionId=31) and it's in the aspnet org.

Reply:
Your release pipeline is also in dnceng-internal so you should be able to use the 'DevDiv - VS package feed' service connection.

Reply:
Clicked on the link and nothing showed. Also, while your release is in dnceng (or devdiv) the same SC should work. Have you tried? Feel free to ping me offline if needed

Reply:
Resolved. Next run of the pipeline will test things out.